native_client/stt.cc

```diff
@@ -167,6 +176,18 @@ StreamingState::processAudioWindow(const vector<float>& buf)
   pushMfccBuffer(mfcc);
 }
 
+void
+StreamingState::flushBuffer()
+{
+  // Flush audio buffer
+  processAudioWindow(audio_buffer_);
+
+  // Process final batch
+  if (batch_buffer_.size() > 0) {
+    processBatch(batch_buffer_, batch_buffer_.size()/model_->mfcc_feats_per_timestep_);
+  }
+}
+
```

Comment:
Maybe rework `finalizeStream` instead of creating a new method? You can rename it to `flushBuffers` and then add a `addZeroMfccVectors=false` parameter which is then set to true when called by `finishStream` and `finishStreamWithMetadata`.